.github/workflows/pull.yml

```diff
@@ -100,6 +100,25 @@ jobs:
           
           Consider insights from discarded methods, even if they weren't the final solution.
 
+        PR_DESCRIPTION__ADD_ORIGINAL_USER_DESCRIPTION: false
+        PR_DESCRIPTION__GENERATE_AI_TITLE: true

```

Comment:
Is the title generated? curious since the title looks like the same suggested by the summary.